### PR TITLE
Svace fix

### PR DESCRIPTION
**Pull request by jeongchanKim**

### CODE REVIEW

**Review by an4967 — DISMISSED** [review]
LGTM

File: apps/examples/wifi_manager_sample/wm_test.c [hunk 1]
```diff
@@ -984,9 +984,6 @@ void wm_auto_test(void *arg)
 
 		printf("[WT] Cycle finished [Round %d]\n", cnt);
 	}
-	printf("[WT] Exit WiFi Manager Stress Test..\n");
```
Comment (sunghan-chang) [inline apps/examples/wifi_manager_sample/wm_test.c:987]: Is this related with SVACE?

File: os/mm/mm_heap/mm_check_heap_corruption.c [hunk 2]
```diff
@@ -152,7 +152,7 @@ int mm_check_heap_corruption(struct mm_heap_s *heap)
 					mm_givesemaphore(heap);
 				}
 				return -1;
-			} else if (node->size != MM_PREV_NODE_SIZE(next)) {
```
Comment (sunghan-chang) [inline os/mm/mm_heap/mm_check_heap_corruption.c:155]: node can't be null so that this is unnecessary.

File: os/mm/mm_heap/mm_check_heap_corruption.c [hunk 3]
```diff
@@ -174,7 +174,7 @@ int mm_check_heap_corruption(struct mm_heap_s *heap)
 				}
 				return -1;
 			} else if (IS_FREE_NODE(node)) {
-				if ((((struct mm_freenode_s *)node)->blink && ((struct mm_freenode_s *)node)->blink->flink != ((struct mm_freenode_s *)node))) {
+				if (node && (((struct mm_freenode_s *)node)->blink && ((struct mm_freenode_s *)node)->blink->flink != ((struct mm_freenode_s *)node))) {
```
Comment (sunghan-chang) [inline os/mm/mm_heap/mm_check_heap_corruption.c:177]: same as previous

File: apps/examples/wifi_manager_sample/wm_test.c (hunk 1 above)
Comment (jeongchanKim) [reply apps/examples/wifi_manager_sample/wm_test.c:987]: Yes. 
WID:10623772 This statement in the source code might be unreachable during program execution.

**Review by sunghan-chang — APPROVED** [review]
